App/ApiRest/Initial.go

```
@@ -0,0 +1,6 @@
+package ApiRest
+
+
+func TurnOn(){
+	LoadServer()
```

This call ignores the error. It may fail and no one would ever know...